demosplan/DemosPlanCoreBundle/Logic/Procedure/ProcedureToLegacyConverter.php

```diff
@@ -102,16 +103,23 @@ protected function convertOrgasToLegacy(Procedure $procedure): array
     {
         $nonPlanningOfficeOrganisationIds = $this->procedureRepository->getInvitedOrgaIds($procedure->getId());
         $planningOfficeIds = $this->procedureRepository->getPlanningOfficeIds($procedure->getId());
-        $isCustomerMasterBlueprint = $procedure->isCustomerMasterBlueprint();
+        $isCustomerMasterBlueprint =
+            $procedure->getId() === $procedure->getCustomer()?->getDefaultProcedureBlueprint()?->getId();
         $planningOfficeOrganisations = collect($procedure->getPlanningOffices())
-            ->transform(static fn(Orga $orga) => [
+            ->transform(static fn (Orga $orga) => [
                 'ident'     => $orga->getId(),
                 'name'      => $orga->getName(),
                 'nameLegal' => $orga->getName(),
             ])->all();
         $dataInputOrgaIds = $procedure->getDataInputOrgaIds();
         $authorizedUserIds = $procedure->getAuthorizedUserIds();
 
+        // T34551 changed the relation of procedure to customer.
+        // previously the procedure->customer relation was null Except for the default-customer-blueprint.
+        // Now only the customer holds the relation to its default-blueprint.
+        $customerToLegacy = $procedure->getId() === $procedure->getCustomer()?->getDefaultProcedureBlueprint()?->getId() ?
```

Comment:
use `isCustomerMasterBlueprint`

Reply:
restored in 6f731c5800aeab34e5de2da82e2fb5c1a96e133c